### PR TITLE
Add route to handle get practitioner details request

### PR DESCRIPTION
**Pull request by bibekmaharjan**
## Description

* Add route to handle /get request to get details of the practitioner.
* Add a controller to handle the request.

### CODE REVIEW

File: src/controller/practitioner.ts
Comment (sbimochan) [inline src/controller/practitioner.ts:1]: Rename these files to practitionerController.ts to separate out from model and other files to ease development

Comment (sbimochan) [reply src/controller/practitioner.ts:1]: also applies for other files

File: src/controller/practitioner.ts [hunk 1]
```diff
@@ -139,3 +139,20 @@ export const deletePractitioner = async (req, res) => {
...
+ * @param {Response} res
+ */
+export const getPractitionerDetail = async (req, res) => {
+  try {
+    const { id } = req.params;
+
+    const response = await Practitioner.findByPk(id);
+    res.send(response);
+  } catch (err) {
+    res.status(500).send({ message: err.message });
```
Comment (sbimochan) [inline src/controller/practitioner.ts:156]: what if it's a not found and 404? does it come in catch block?

File: src/routes/practitioner.routes.ts [hunk 2]
```diff
@@ -12,11 +19,13 @@ export default (app) => {
     next();
   });
 
-  app.get('/practitioners', getPractitioner);
+  app.get('/practitioners', verifyJwtToken, getPractitioner);
 
-  app.post('/practitioners', [upload.single('userImg'), schema(practitioner)], addPractitioner);
+  app.post('/practitioners', [upload.single('userImg'), schema(practitioner), verifyJwtToken], addPractitioner);
```
Comment (sbimochan) [inline src/routes/practitioner.routes.ts:24]: what's upload.single('userImg') ?

Comment (bibekmaharjan) [reply src/routes/practitioner.routes.ts:24]: upload.single('userImg') is a middleware function provided by the multer library, which is used for handling file uploads.

**Pull request merged**